docs/source/metrics.rst

```diff
@@ -105,6 +104,13 @@ This metrics API is independent of PyTorch Lightning. Metrics can directly be us
     # total accuracy over all validation batches
     total_valid_accuracy = valid_accuracy.compute()
 
+.. warning::
+
+    Metrics have internal states that keeps track of the data seen so far. Too
+    not mix metric states across training, validation and testing it is
+    highly recommended to re-initialize the metric per mode as shown in the
+    examples above.
+
```

Comment:
I think there is a spelling mistake here, might be better to reword like below:

Metrics contain internal states that keep track of the data seen so far. Do not mix metric states across training, validation and testing. It is highly recommended to re-initialize the metric per mode as shown in the examples above.

Reply:
thanks, changed it to your suggestion 👍